web/src/engine/providers/MangaPlugin.ts

```diff
@@ -76,6 +79,20 @@ export class DecoratableMangaScraper extends MangaScraper {
     public FetchImage(_page: Page, _priority: Priority, _signal: AbortSignal): Promise<Blob> {
         throw new Error();
     }
+
+    public async FilterChapters(list: Chapter[]): Promise<Chapter[]> {
+        return list.filter(
+            (obj, index) =>
+                list.findIndex((item) => item.Identifier === obj.Identifier) === index
```

Comment:
This will just take the first manga with the duplicated identifier in the list, it does not cover the case that the first manga may have the title 'Error' and the second manga may have the title 'My little Sheep'. Obviously, in this case the second manga is the one that should be picked.

Reply:
```typescript
export function GetUniqueMediaItems<T extends IMediaItem>(items: Array<T>): Array<T> {
    return [...items]; // TODO: Only get unique entries by `Identifier` property
}

Reply:
(its chapters here but i see your point)

> Obviously, in this case the second manga is the one that should be picked.

Obviously, but would that "rule" you made about the title be the same everywhere? Also, chapter title may have "Error" in the title just because well, its part of the real title.

The point of this filtering is to be generic, and will be fine 95% of the time. Most of the time it wont be necessary, and when its necessary chapters tends to be named with the same name. I dont think we will find another rule that wont cause a problem on one website because its needed on another.  If you have more idea about filtering rules i am open to discussions ;)

For such edges cases of and edge cases, dev can override the Filter methods 🤷 

The other solution is to filter identical ID + identical title in this, but then the UI would have to be reworked to work with ID+title so there is no bug with only identical ID

Reply:
I have no suggestion how to determine the best fitting of the duplicates.

Another solution that comes to mind would be to keep the duplicates. the object instances itself are different, even if the ID property is the same. So programatically they can be distinguished. If the UI needs a unique identifier, maybe adding a hash method or a symbol field to identify the object instance may solve the problem...

Reply:
The hash solution is elegant. Generating one is not a big deal, same for storage but its putting more work on the ui (i mean it need reworking )

Reply:
Ok, I think i'll change the filter to (same id + same title) its already a simple solution worth to be validated which wont hinder further fixes. I'll let the issue opened.

Reply:
Hum, should i change the filtering for (same iD+ same title)? And hope UI can filter dupes using instance?